[2025-05-24T12:23:57.558+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_transaction_data_and_upload_to_hdfs.run_hive_command scheduled__2025-05-23T00:00:00+00:00 [queued]>
[2025-05-24T12:23:57.572+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_transaction_data_and_upload_to_hdfs.run_hive_command scheduled__2025-05-23T00:00:00+00:00 [queued]>
[2025-05-24T12:23:57.573+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-24T12:23:57.591+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): run_hive_command> on 2025-05-23 00:00:00+00:00
[2025-05-24T12:23:57.600+0000] {standard_task_runner.py:57} INFO - Started process 363 to run task
[2025-05-24T12:23:57.604+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fetch_transaction_data_and_upload_to_hdfs', 'run_hive_command', 'scheduled__2025-05-23T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmpivuvyy2k']
[2025-05-24T12:23:57.609+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask run_hive_command
[2025-05-24T12:23:57.677+0000] {task_command.py:415} INFO - Running <TaskInstance: fetch_transaction_data_and_upload_to_hdfs.run_hive_command scheduled__2025-05-23T00:00:00+00:00 [running]> on host 1190334ace1e
[2025-05-24T12:23:57.947+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_transaction_data_and_upload_to_hdfs' AIRFLOW_CTX_TASK_ID='run_hive_command' AIRFLOW_CTX_EXECUTION_DATE='2025-05-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-23T00:00:00+00:00'
[2025-05-24T12:23:57.949+0000] {logging_mixin.py:151} INFO - üîç Checking if metastore_db exists...
[2025-05-24T12:23:58.273+0000] {logging_mixin.py:151} INFO - ‚ùå metastore_db does NOT exist. Creating it and initializing schema...
[2025-05-24T12:23:58.726+0000] {logging_mixin.py:151} INFO - ‚öôÔ∏è Initializing Hive metastore schema...
[2025-05-24T12:24:05.776+0000] {logging_mixin.py:151} INFO - üöÄ Running Hive script...
[2025-05-24T12:24:21.148+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-24T12:24:21.161+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=fetch_transaction_data_and_upload_to_hdfs, task_id=run_hive_command, execution_date=20250523T000000, start_date=20250524T122357, end_date=20250524T122421
[2025-05-24T12:24:21.245+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-24T12:24:21.272+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-24T13:30:06.695+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_transaction_data_and_upload_to_hdfs.run_hive_command scheduled__2025-05-23T00:00:00+00:00 [queued]>
[2025-05-24T13:30:06.707+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_transaction_data_and_upload_to_hdfs.run_hive_command scheduled__2025-05-23T00:00:00+00:00 [queued]>
[2025-05-24T13:30:06.707+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-24T13:30:06.722+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): run_hive_command> on 2025-05-23 00:00:00+00:00
[2025-05-24T13:30:06.730+0000] {standard_task_runner.py:57} INFO - Started process 328 to run task
[2025-05-24T13:30:06.733+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fetch_transaction_data_and_upload_to_hdfs', 'run_hive_command', 'scheduled__2025-05-23T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmpbwcxgf4_']
[2025-05-24T13:30:06.736+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask run_hive_command
[2025-05-24T13:30:06.806+0000] {task_command.py:415} INFO - Running <TaskInstance: fetch_transaction_data_and_upload_to_hdfs.run_hive_command scheduled__2025-05-23T00:00:00+00:00 [running]> on host 1be44660cb93
[2025-05-24T13:30:07.084+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_transaction_data_and_upload_to_hdfs' AIRFLOW_CTX_TASK_ID='run_hive_command' AIRFLOW_CTX_EXECUTION_DATE='2025-05-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-23T00:00:00+00:00'
[2025-05-24T13:30:07.085+0000] {logging_mixin.py:151} INFO - üîç Checking if metastore_db exists...
[2025-05-24T13:30:07.377+0000] {logging_mixin.py:151} INFO - ‚ùå metastore_db does NOT exist. Creating it and initializing schema...
[2025-05-24T13:30:06.609+0000] {logging_mixin.py:151} INFO - ‚öôÔ∏è Initializing Hive metastore schema...
[2025-05-24T13:30:14.587+0000] {logging_mixin.py:151} INFO - üöÄ Running Hive script...
[2025-05-24T13:30:28.284+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-24T13:30:28.297+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=fetch_transaction_data_and_upload_to_hdfs, task_id=run_hive_command, execution_date=20250523T000000, start_date=20250524T133006, end_date=20250524T133028
[2025-05-24T13:30:28.335+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-24T13:30:28.362+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
